modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -2010,7 +2010,7 @@
     /**
      * Determines whether to use the experimental sql, calcite based, engine.
      */
-    @SystemProperty(value = "Determines whether to use the experimental sql, calcite based, engine.",
+    @SystemProperty(value = "Determines whether to use the experimental sql, calcite based, engine",
```

Comment:
Please revert this (this property will be removed by another patch, to avoid merge conflicts)